src/main/java/ro/unibuc/hello/dto/CreateTask.java

```diff
@@ -0,0 +1,20 @@
+package ro.unibuc.hello.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
```

Comment:
`@Data` instead of `@Getter` + `@Setter`?

Reply:
I think a class should be as simple as possible, so if there is no need for toString or equals&hashcode, i don't see the point of using @Data

Reply:
Fair enough. RIP @Data probably they get tagged in so many places =))))))))))))

Reply:
Agree :laughing: